### ISSUE TITLE
Plans to support Azure SQL DB?

### ISSUE THREAD

**Issue by jaspertandy** [TablePlus/TablePlus#3236]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):

MSSQL. I've tried all versions of this server and none of them work. Azure Data Studio tells me that the version of the server is:

Azure SQL DB 12.0.2000.8

I was under the impression that this would be MSSQL compatible but it doesn't appear to be. I know very little about this environment as I use it in a readonly capacity. Obviously if it's essentially an entirely new connection type then I'm not expecting it'll be supported!

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):

538

3. The steps to reproduce this issue:

Create a new connection for an Azure SQL DB. When testing, see the following error:

![screenshot--2024-03-01 at 17 45 09](https://github.com/TablePlus/TablePlus/assets/350834/bea8f958-5fde-4c46-a13a-6904d105f134)

**Comment by huyphams** [comment]
Hi @jaspertandy could you try this?
https://stackoverflow.com/questions/16047905/how-do-i-connect-to-sql-azure-from-nodejs-tedious

**Comment by huyphams** [comment]
The username should be:
SomeUser@someazureserver.database.windows.net

**Comment by jaspertandy** [comment]
Changing to that is getting me an authentication error rather than a version error, so I suspect that it's to do with how I'm extracting the server name from the hostname I was given - thanks I'll go back to the vendor and see if they can help me out.

**Comment by huyphams** [comment]
Yes, because I read the driver documents and it said that Azure SQL is supported.
